packages/core/.eslintrc.js

```diff
@@ -59,7 +59,7 @@ const commonExtends = ["plugin:react/recommended", "plugin:react-hooks/recommend
 module.exports = {
   overrides: [
     {
-      files: ["*.jest.js", "jest.init.js"],
+      files: ["*.test.js", "jest.init.js"],
```

Comment:
it does not take into consideration `*.test.tsx` or `*.test.ts`
what is the meaning of it not catching it?